src/qmapshack/gis/search/CSearch.cpp

```diff
@@ -507,6 +507,8 @@ QMap<searchProperty_e, QString> CSearch::initSearchPropertyMeaningMap()
     map.insert(eSearchPropertyGeneralDate, tr("searches the Date"));
     map.insert(eSearchPropertyGeneralComment, tr("searches the Comment"));
     map.insert(eSearchPropertyGeneralDescription, tr("searches the Description"));
+    map.insert(eSearchPropertyGeneralKeywords, tr("searches the Keywords set"));
```

Comment:
"Set Tags"
"Keywords"  
The wording should be more unified.

Reply:
In my opinion tags contains both keywords and ratings.

Reply:
ah, ok. Good to know for translation. 

but change it to "searches the Keywords". We know it's a set but the user does not.

Reply:
Ah, the "set" shouldn't mean the set as a union of elements but as the verb. However I agree that it should go.